package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "start": "./entry.sh",
-    "build": "coffee -c src",
+    "build": "coffee -c src && webpack -p",
```

Comment:
Can we use `coffee-loader` in webpack to do the coffee compilation please, rather than having two separate build steps